crates/libs/implement/src/tests.rs

```diff
@@ -0,0 +1,130 @@
+//! These tests are just a way to quickly run the `#[implement]` macro and see its output.
+//! They don't check the output in any way.
+//!
+//! This exists because of some difficulties of running `cargo expand` against the `#[implement]`
```

Comment:
Maybe just a comment about how to run a test and see the output - for example I run this:

```
 cargo test -p windows-implement --tests simple_type
```

But cargo test does not automatically display the `stdout` for `println`.

Reply:
e.g. 

```
cargo test -p windows-implement --tests simple_type -- --nocapture
```